docs/markdown/Rust-module.md

```diff
@@ -18,7 +18,7 @@ like Meson, rather than Meson work more like rust.
 
 ## Functions
 
-### test(name: string, target: library | executable, dependencies: []Dependency)
+### test(name: string, target: library | executable, dependencies: []Dependency, link_with: []targets)
```

Comment:
I continue to find this an unpleasant and ugly way to document anything.

e.g. https://mesonbuild.com/Rust-module.html#bindgen-input-string-buildtarget-string-buildtarget-output-string-include_directories-include_directories-string-c_args-string-args-string-dependencies-dependency is totally unreadable and also impossible to stably link to.

It's also an out-and-out lie, because `rust_mod.test(name: 'foobar', target: mytarget)` isn't a valid invocation approach, but "dependencies" *is* a kwarg rather than a typed_pos_arg, and this documentation asserts a positive statement that they are all of them kwargs.

No one is using this information, because they literally cannot (figure out how to read it).

This is not a blocking review comment, just an existing issue that I thought I'd mention while I'm here.

Reply:
I know you don't like it, but at this point the only realistic way to make all of the module docs consistent is for someone to convert them to the yaml format and generate them them that way, because any other attempt to standardize them just seems like a waste of energy :/